src/main/java/com/ibm/watson/developer_cloud/relationship_extraction/v1/RelationshipExtraction.java

```diff
@@ -96,5 +99,15 @@ public Dataset getDataset() {
   public void setDataset(final Dataset dataset) {
     this.dataset = dataset;
   }
+  
+  /**
+   * Sets the return type.
+   * 
+   * @param rt the new rt
+   */
+  
+  public void setReturntype(final String rt) {
```

Comment:
`setReturnType` not `setReturntype` and make sure you use `returnType` instead of `rt`.

Can you clarify the different returns types in the javadoc or maybe create an `Enum` class with them ?